test/integration/model.test.js

```diff
@@ -480,155 +480,157 @@ describe(Support.getTestDialectTeaser('Model'), () => {
       });
     });
 
-    it('should allow the user to specify indexes in options', async function () {
-      const indices = [{
-        name: 'a_b_uniq',
-        unique: true,
-        method: 'BTREE',
-        fields: [
-          'fieldB',
-          {
-            attribute: 'fieldA',
-            collate: dialect === 'sqlite' ? 'RTRIM' : 'en_US',
-            order: dialect === 'ibmi' ? '' : `${isMySQL8 ? 'ASC' : 'DESC'}`,
-            length: 5,
-          },
-        ],
-      }];
+    if (dialect !== 'yugabyte'){ // Yugabyte doesn't support creation of index with concurrently option
+      it('should allow the user to specify indexes in options', async function () {
```

Comment:
why don't simply use something like : `current.dialect.name="yugabytedb" ? it.skip : it` as used above. That will just get reflected as only one line change in this file.

Reply:
Yah, we can do that, actually this change I made previously till then I was not using it.skip as I thought it is not a good way now as it was giving error to me in my editor but that is actually resolved then so I will make changes to this test also.